pkg/bootstrap/docker/up.go

```diff
@@ -920,6 +924,21 @@ func (c *ClientStartConfig) ImportTemplates(out io.Writer) error {
 	return c.importObjects(out, "kube-system", adminTemplateLocations)
 }
 
+func useAnsible(imageVersion string) bool {
```

Comment:
We should likely not rely on the imageVersion argument ... it could be anything like a commit id, or simply "latest". We should try to get the version from the openshift binary in the image:
https://github.com/openshift/origin/blob/master/pkg/bootstrap/docker/openshift/helper.go#L618
(Just make that method public)